extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -55,6 +55,7 @@ tags: {  # <amp-video>
   }
   attrs: { name: "placeholder" }
   attrs: { name: "poster" }
+  attrs: { name: "crossorigin" }
```

Comment:
This allows *any* value for the crossorigin attribute. If you want to only allow empty, you can use 

```
attrs: {
  name "crossorigin"
  value: ""
}`
```

This will also allow `crossorigin=` and `crossorigin=crossorigin` as they are semantically identical.